ui/components/multichain/network-list-menu/network-list-menu.js

```diff
@@ -111,7 +112,7 @@ export const NetworkListMenu = ({ onClose }) => {
                   dispatch(
                     showModal({
                       name: 'CONFIRM_DELETE_NETWORK',
-                      target: network.id || network.chainId,
+                      target: network.id || network.caipChainId, // should this be turned into hex?
```

Comment:
check this

Reply:
it expects networkConfigurationId at NetworkController.removeNetworkConfiguration()